i18n/en-US.yml

```diff
@@ -387,11 +420,10 @@ common:
   time:
     # Use ordered placeholders for the departure-arrival string
     # (this will accommodate right-to-left languages by swapping the order in this string).
-    departureArrivalTimes: "{startTime, time, short}—{endTime, time, short}"
```

Comment:
I think we started to favor placing the time formatting options this way, and it simplifies the code. What was the reason for reverting to the previous approach?

Reply:
I removed the formatting here bc I figured that the previous component `FormattedTimeRange` was probably unnecessary and could be replaced with direct calls to `common.time.departureArrivalTimes` w/ FormattedTime values provided. This also removed the need for moment.js to interpret the intinerary.startTime and endTime values before providing them to the FormattedMessage. 

This change looked like this: 
![image](https://user-images.githubusercontent.com/63798641/137781907-1c2179c8-f176-4d45-bfd2-ae7b10044a07.png)

to this: 
![image](https://user-images.githubusercontent.com/63798641/137782635-bdf667ff-5188-4e25-b822-c7c52c46afc7.png)

Reply:
Yes thank you, that looks much better.